.run/Main.run.xml

```diff
@@ -5,7 +5,7 @@
     <option name="MAIN_CLASS_NAME" value="digital.slovensko.autogram.Main" />
     <module name="autogram" />
     <option name="PROGRAM_PARAMETERS" value="--url=autogram://listen?protocol=http&amp;host=localhost&amp;port=37200&amp;origin=*&amp;language=sk" />
-    <option name="VM_PARAMETERS" value="--add-exports javafx.graphics/com.sun.javafx.tk=ALL-UNNAMED" />
+    <option name="VM_PARAMETERS" value="--add-exports jdk.crypto.cryptoki/sun.security.pkcs11.wrapper=ALL-UNNAMED --add-opens java.base/java.security=ALL-UNNAMED --add-opens jdk.crypto.cryptoki/sun.security.pkcs11=ALL-UNNAMED" />
```

Comment:
tento súbor je pre maven a teda asi pre intellij. Pre VSCode ešte asi ja upravím, zatiaľ sa mi to nepodarilo na správnom mieste.

Reply:
Ano toto je intellij - ktory inak ide mimo maven tiez.